torch/ao/quantization/observer.py

```diff
@@ -1902,10 +1902,18 @@ def convert(self, model: torch.fx.GraphModule, observer_node: Node):
             else:
                 scale, zero_point = self.calculate_qparams()
                 scale_node = create_getattr_from_value(
-                    model, model.graph, "_scale", scale
+                    model,
+                    model.graph,
+                    "_scale",
+                    scale,
+                    scale.device,
```

Comment:
value could be a non-Tensor I think? according to `create_getattr_from_value` L274, so should this be the device from model?